fractions/microprofile/microprofile-metrics/src/main/java/org/wildfly/swarm/microprofile/metrics/runtime/MetricsHttpHandler.java

```diff
@@ -60,7 +61,8 @@ public void handleRequest(HttpServerExchange exchange) throws Exception {
 
         String method = exchange.getRequestMethod().toString();
         HeaderValues acceptHeaders = exchange.getRequestHeaders().get(Headers.ACCEPT);
-        metricsHandler.handleRequest(requestPath, method, acceptHeaders.stream(), (status, message, headers) -> {
+        Stream<String> acceptHeadersStream = acceptHeaders != null ? acceptHeaders.stream() : null;
+        metricsHandler.handleRequest(requestPath, method, acceptHeadersStream, (status, message, headers) -> {
```

Comment:
TBH, I really don't think `Stream` is an acceptable parameter type or return type in a public API (for the same reason as `Iterator`), but SmallRye decided otherwise.